### PR TITLE
[BL-56] Add schema names to converter

### PR DESCRIPTION
**Pull request by jkmar**
Converter now generates const with schema ids.
Additionally schema id type is changed from string to separate type "SchemaID"

### CODE REVIEW

**Comment by morrisson** [comment]
What is "schema names"?

**Comment by jkmar** [comment]
@morrisson 
Schema names was list of schema ids, but I've changed this pr to include SchemaID as separate type.

File: extension/goext/schemas.go [hunk 1]
```diff
@@ -20,6 +20,9 @@ import (
 	"errors"
 )
 
+// SchemaID is a type for schema ID
+type SchemaID string
```
Comment (marcin-ptaszynski) [inline extension/goext/schemas.go]: What's the benefit of creating new type in this case?

Comment (p-kozlowski) [reply extension/goext/schemas.go]: type safety

**Comment by jkmar** [comment]
@marcin-ptaszynski 
Could you take a look at this pr again?

**Pull request merged**